src/methods/modular_programs/easy/q4/ReturnChange.java

```diff
@@ -0,0 +1,31 @@
+package methods.modular_programs.easy.q4;
+
+/*
+Write a Java Program that initializes three double variables and passes them to a method called valueAfterCalc to
+return the number values after the decimal point. (e.g. if the  number is 14.5673, return 0.5673). If the variables are
```

Comment:
```suggestion
return the number values after the decimal point. (e.g. if the number is 14.5673, return 0.5673). If the variables are
```

Reply:
This just removes an extra space in the question text (make sure to also update the website